rlapy/tests/matmakers.py

```diff
@@ -29,3 +30,37 @@ def simple_mat(n_rows, n_cols, scale, rng):
     RA *= damp
     A_bad = QA @ RA
     return A_bad
+
+
+def exponent_spectrum(n_rows, n_cols, spectrum: Union[int, np.ndarray], rng, spectrum_param, factors=False):
+    rng = np.random.default_rng(rng)
+    if isinstance(spectrum, int):
+        # Is this faster than np.empty(spectrum, float)?
+        spectrum = rng.random(size=(spectrum,))
```

Comment:
``np.empty(spectrum, float)`` would be faster than calling ``rng.random``. But the speed difference is negligible.